go.mod

```diff
@@ -1,9 +1,18 @@
 module github.com/hashicorp/gorm-cursor-paginator
 
-go 1.16
+go 1.23
```

Comment:
Is 1.23 okay, or should we be using 1.22 still for compatibility with the platform?

Reply:
yeah.. i just saw that our tooling is still not supporting 1.23. let me change that.